public/index.html

```diff
@@ -85,6 +85,7 @@
 <!-- Bootstrap core JavaScript
 ================================================== -->
 <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
-<script type="text/javascript" src="components/requirejs/require.js" data-main="js/bootstrap.js"></script>
+<script type="text/javascript" src="../components/requirejs/require.js" data-main="../build/js/main.min.js"></script>
```

Comment:
How this wil work before build? There is no file main.min.js in src`s, so it wouldn't work. Am I right?